src/Compilers/CSharp/Portable/Symbols/PublicModel/AssemblySymbol.cs

```diff
@@ -55,6 +55,12 @@ INamedTypeSymbol IAssemblySymbol.ResolveForwardedType(string fullyQualifiedMetad
             return UnderlyingAssemblySymbol.ResolveForwardedType(fullyQualifiedMetadataName).GetPublicSymbol();
         }
 
+        ImmutableArray<INamedTypeSymbol> IAssemblySymbol.GetForwardedTypes()
+        {
+            return UnderlyingAssemblySymbol.GetAllTopLevelForwardedTypes().Select(t => t.GetPublicSymbol()).
+                   OrderBy(t => t.ToDisplayString(SymbolDisplayFormat.QualifiedNameArityFormat)).AsImmutable();
```

Comment:
>OrderBy [](start = 19, length = 7)

Nit: consider putting the `.` on this line. Having it on the previous line and this line starting with what appears to be receiver-less invocation is confusing.

Reply:
I am coming from school that leaves operators on the previous line.

---
In reply to: [426949277](https://github.com/dotnet/roslyn/pull/44307#discussion_r426949277) [](ancestors = 426949277)